spec/suites/layer/marker/MarkerSpec.js

```diff
@@ -159,7 +159,7 @@ describe("Marker", function () {
 			} else {
 				expect(marker._icon.src).to.contain('marker-icon.png');
 			}
-			expect(marker._icon.parentNode).to.be(map._panes.markerPane);
+			expect(marker._icon.parentNode.parentNode).to.be(map._panes.markerPane);
```

Comment:
> Parent of parent is an overkill.
> It's obvious that something is wrong in marker design.

_Originally posted by @johnd0e in https://github.com/Leaflet/Leaflet/pull/6248#r412835549_